### ISSUE TITLE
Clean up Phidget reading in

### ISSUE THREAD

**Issue by pdeperio** [PMT-Test-Facilities/ptf-analysis#21]
Vectorize and make configurable the reading in of Phidget data, similar to [Thomas' enhancement here](https://github.com/PMT-Test-Facilities/ptf-analysis/pull/18/files/7d82e9444bbc4b48953eed1a1a15e76f5f9527ab..8aed8288848915a115b2826135498784f1589d45) from https://github.com/PMT-Test-Facilities/ptf-analysis/pull/18

**Comment by vgousyleblanc** [comment]
Yes I agree it is a excellent idea not to have too many copy pasting to do. I had orginally done it the simplest way possible because I was not super confortable with this new way. I will add it to the list of things to do !

**Comment by vgousyleblanc** [comment]
So I don't know how much different this will now be, I looked at the phidget and to get the data out all we need to do is select the channel of the phidget and select the data associated with it (either magnetic field or acceleration). So in the end, I am not entirely sure of what this enhancement would be...
